### PR TITLE
Core RP preview camera background color

### PR DESCRIPTION
**Pull request by martint-unity**
---
### Purpose of this PR
Fixing the issues people are experiencing with a darker background for the previews.
https://fogbugz.unity3d.com/f/cases/1297865/

Also just moving the gizmo color into a sub menu and make sure we can reset gizmo color

---
### Testing status
Opened up both URP and HDRP and created Prefabs and checking previews that the color is the default gray.

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.core/Runtime/Utilities/CoreRenderPipelinePreferences.cs [hunk 1]
```diff
@@ -27,26 +36,61 @@ public static Color volumeGizmoColor
             }
         }
 
+        public static Color previewBackgroundColor
```
Comment (madsnyholm) [inline com.unity.render-pipelines.core/Runtime/Utilities/CoreRenderPipelinePreferences.cs:39]: This Preview background color cannot be a User Preference.
See discussion https://unity.slack.com/archives/C0A9NNY9W/p1608024382240300?thread_ts=1607848679.236200&cid=C0A9NNY9W

**Review by madsnyholm — CHANGES_REQUESTED** [review]
Please remove the Preview background color from Preferences and the Settings Provider. It needs to be a const color, since there are no decencies setup that will ensure all assets are reimported if the background color changes. 
See discussion https://unity.slack.com/archives/C0A9NNY9W/p1608024382240300?thread_ts=1607848679.236200&cid=C0A9NNY9W

**Review by madsnyholm — APPROVED** [review]
Looks good.

**Review by JulienIgnace-Unity — APPROVED** [review]

**Review by sebastienlagarde — APPROVED** [review]

**Review by RSlysz — APPROVED** [review]
lgtm

**Review by TomasKiniulis — COMMENTED** [review]
Taking a look on HDRP side. If I update an older project where preview was black then prefab still holds black background even after doing a reimport, same goes to Fbx and material has to be clicked to change background. Would it be possible to have this fixed in the same PR? 

upgradedProjectConvert

**Comment by martint-unity** [comment]
@TomasKiniulis You will have to do a Re-import all for this to take affect on old projects.

**Review by erikabar — APPROVED** [review]
Tested Material, Mesh, Prefab, Model Project preview and Inspector preview backgrounds, the issue is fixed.

**Review by TomasKiniulis — APPROVED** [review]
Approving this, although I see a potential issue where users might be confused why background colors vary between old assets and newly created ones on project upgrade without "Reimport All", but there doesn't seem to be other way to resolve this

**Review by phi-lira — APPROVED** [review]

**Pull request merged**